index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="green">
+      <h1>Hello Microverse!</h1>
+    </header>
+    <section>
+      <h1>Section 1</h1>
+      <p id="title">Section 1</p>
+    </section>
+    <footer>
+      <h3 id="title">h3 element</h3>
```

Comment:
Kindly change the ```id='title'``` for the h3 tag. Element IDs should be unique within the entire document.

Reply:
Kindly change the `id='title'` for the h3 tag because using duplicate ids may break the accessibility and cause errors in the future. To fix this problem change the duplicate id value to ensure each id is unique. To get more information about why duplicate ids are bad, see this [article](https://dequeuniversity.com/rules/axe/4.1/duplicate-id-active).